2.3.1 release announcement (#257)
mixxxdj/website: content/news/XXXX-XX-XX-mixxx-2-3-1-released.md

title: Mixxx 2.3.1 released
authors: Daniel Schürmann
tags: 2.3, 2.3.1, release announcement

> status: draft

Reviewer: I guess the draft status should also be removed before merging?

Reviewer 2: No, from the README:
> Let the status on draft and don't add a date field. When the PR is merged, GitHub actions will automatically rename the file, add the appropriate date and remove the draft status.

comments: yes

> We are happy to announce first bugfix release of [Mixxx 2.3.1](https://github.com/mixxxdj/mixxx/releases/tag/2.3.1)!

Reviewer 2: ```suggestion
We are happy to announce the release of [Mixxx 2.3.1](https://github.com/mixxxdj/mixxx/releases/tag/2.3.1)!
```

Reviewer 2: "first release of Mixxx 2.3.1" is odd; it implies there will be other releases of Mixxx 2.3.1.

> Head over to the [download page] ({filename}/pages/download.md#stable) and download it on Windows, macOS and Linux.

Reviewer: ```suggestion
Head over to the [download page]({filename}/pages/download.md#stable) and download it on Windows, macOS and Linux.
```
The space breaks the transformation into a link.

> This release comes with new and updated controller mappings, fixes packaging issues and includes some bugfixes.

Reviewer 2: ```suggestion
This release fixes a handful of bugs and packaging issues plus adds several controller mappings.
```

> Mixxx now includes mappings for the [Numark DJ2GO2 Touch](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_dj2go2_touch.html) controller and the [Numark Mixtrack Pro FX](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_mixtrack_pro_fx.html) controller. The [Behringer DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html), the [Hercules Inpulse 300](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_inpulse_300.html) mapping, the [Denon MC7000](https://manual.mixxx.org/2.3/en/hardware/controllers/denon_mc7000.html) mapping and the [Traktor S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html) mapping have been improved with Mixxx 2.3.

Reviewer 2: ```suggestion
Mixxx now includes mappings for the [Numark DJ2GO2 Touch](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_dj2go2_touch.html) controller and the [Numark Mixtrack Pro FX](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_mixtrack_pro_fx.html) controller. The [Behringer DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html), the [Hercules Inpulse 300](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_inpulse_300.html) mapping, the [Denon MC7000](https://manual.mixxx.org/2.3/en/hardware/controllers/denon_mc7000.html) mapping and the [Traktor Kontrol S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html) mapping have been improved since Mixxx 2.3.0.
```
